extension/src/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { finder } from '@medv/finder'
```

Comment:
Not sure I'm happy with the selectors this generates. They seem very brittle. I might try some alternative in the future.